configure.ac

```diff
@@ -3019,17 +3023,9 @@ if test x$host_win32 = xno; then
 	])
 
 	if test x$host_darwin = xno; then
-		AC_MSG_CHECKING(for struct flock64)
-		AC_TRY_COMPILE([
-			#include <fcntl.h>
-		], [
-			struct flock64;
-		],[
-			AC_MSG_RESULT(yes)
-			AC_DEFINE(HAVE_FLOCK64, 1, struct flock64)
-		], [
-			AC_MSG_RESULT(no)
-		])
+		# We can't match autotools with structs, so try using a member we know will
```

Comment:
I don't understand the comment.

Reply:
comment dated back to an older patch that matched on a structure member, it's senseless now that it matches on type and I'll remove it